wallet/src/de/schildbach/wallet/payments/RequestPaymentRequestTask.java

```diff
@@ -130,102 +122,6 @@ protected void error(Exception x, final int messageResId, final Object... messag
         }
     }
 
-    public final static class BluetoothRequestTask extends RequestPaymentRequestTask {
```

Comment:
Doesn't seem to be used.

Reply:
Bluetooth features were part of the app in prior versions, though they were probably not used by many.  Like NFC it was an alternate way to send payments or payment requests to another user's device.